k8s/charts/openebs/values.yaml

```diff
@@ -32,10 +32,11 @@ apiserver:
   healthCheck:
     initialDelaySeconds: 30
     periodSeconds: 60
-
+  defaultStorageConfig:
+    enabeled: "false"
```

Comment:
spell check.

Also, can you move this `defaultStorageConfig` section as its own components.

Reply:
Related to this change, you need to make the change in the maya-apiserver deployment. Also need to add this flag into the README.

Reply:
Also add `OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG` env in Maya-apiserver deployment yaml file and set the value as `false` via this tag.
```
defaultConfigs:
    StorageConfig : 
         enabled: "false"
```

Reply:
@kmova is `defaultStorageConfig` is true by default ?